components/countdown.tsx

```diff
@@ -6,18 +6,21 @@ const Countdown: FC = () => {
     const day = props.days > 1 ? 'days' : 'day'
     const hour = props.hours > 1 ? 'hours' : 'hour'
     const minutes = props.minutes > 1 ? 'minutes' : 'minute'
+    // const seconds = props.seconds > 1 ? 'seconds' : 'second'
     return (
       <span>
-        {/* {props.days} {day + " "} */}
-        {/* {props.hours} {hour + " "}
-        {props.minutes} {minutes} */}
+        {props.days} {day + " "}
+        {props.hours} {hour + " "}
+        {props.minutes} {minutes}
+        {/* {props.seconds} {seconds} */}
       </span>
     )
   }
 
   return (
     <div className='text-center text-xl mx-auto py-2 bg-wisteria text-black select-none font-medium'>
-      <span>Applications now open! </span>
+      <span>Hacking begins in </span>
+      <Count date={1616864400000} renderer={renderer} />
```

Comment:
Because it's a little late, change this to reflect the "Devpost submissions close" deadline, make the other requested changes, and submit another PR for the conditional swapping so we can merge this one.